modules/sdk-coin-avaxp/test/unit/avaxp.ts

```diff
@@ -340,18 +342,63 @@ describe('Avaxp', function () {
     });
   });
 
-  describe('Validation', function () {
+  describe('Address Validation', function () {
```

Comment:
nit: we validate more than just addresses inside this block
```suggestion
  describe('Validation', function () {
```

Reply:
done